packages/myst-to-tex/src/index.ts

```diff
@@ -77,6 +77,26 @@ const createGlossaryDefinitions = (tree: Root): Record<string, [string, string]>
       .filter((x) => x.length > 0), // remove empty
   );
 
+const createAcronymDefinitions = (tree: Root): Record<string, [string, string]> =>
+  // Note: Abbreviations contain their resolved values, hence there
+  //       can be many duplicates which will be collapsed when
+  //       creating the dictionary
+  Object.fromEntries(
+    selectAll('abbreviation', tree)
+      .map((node) => {
+        const a = node as Abbreviation;
+        if (!a.title) {
+          return [];
+        }
+        const t = select('text', node) as Text;
```

Comment:
Yeah it makes sense. I agree with your points, little unsure about possible cases here. I have a feeling there are a couple corner situations we are not considering, but let us go with this and see later in case. Once problems occur, we will revisit the logic and our assumptions :)